modules/hub/hub/views.py

```diff
@@ -767,3 +775,21 @@ def post(self):
                 search = {"type": "ip", "text": form.searchText.data}
 
         return redirect(url_for("hub.connections", server=hub_current_server.id, search=search))
+
+class AdminsView(Hub):
+    def get_admins():
+        server_admins = game_models[hub_current_server.id]["ErroAdmin"]
+        admins = [admin for admin in server_admins.query.order_by(server_admins.rank).all()]
+        result = {rank:[admin.ckey for admin in admins if admin.rank == rank] for rank in [admin.rank for admin in admins]}
+        return result
+
+    def get(self):
+        is_json = request.endpoint
```

Comment:
Переменная не соответствует смыслу.

```suggestion
        is_json = (request.endpoint == "hub.admins_json")
```

Reply:
а так можно было?